packages/client-core/src/admin2/components/user/UserTable.tsx

```diff
@@ -0,0 +1,140 @@
+The Original Code is Ethereal Engine.
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import { UserType, userPath } from '@etherealengine/common/src/schema.type.module'
+import { State, getMutableState, useHookstate } from '@etherealengine/hyperflux'
+import { useFind, useSearch } from '@etherealengine/spatial/src/common/functions/FeathersHooks'
+import AvatarImage from '@etherealengine/ui/src/primitives/tailwind/AvatarImage'
+import Checkbox from '@etherealengine/ui/src/primitives/tailwind/Checkbox'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { HiPencil, HiTrash } from 'react-icons/hi2'
+
+import { PopoverState } from '../../../common/services/PopoverState'
+import { AuthState } from '../../../user/services/AuthService'
+import { userHasAccess } from '../../../user/userHasAccess'
+import DataTable from '../../common/Table'
+import { UserRowType, userColumns } from '../../common/constants/user'
+import AccountIdentifiers from './AccountIdentifiers'
+import AddEditUserModal from './AddEditUserModal'
+import RemoveUserModal from './RemoveUserModal'
+
+export default function UserTable({
+  search,
+  skipGuests,
+  selectedUsers
+}: {
+  skipGuests: boolean
+  search: string
+  selectedUsers: State<UserType[]>
+}) {
+  const { t } = useTranslation()
+  const user = useHookstate(getMutableState(AuthState).user)
+
+  const adminUserQuery = useFind(userPath, {
+    query: {
+      isGuest: skipGuests ? false : undefined,
+      $skip: 0,
+      $limit: 20,
+      $sort: {
+        name: 1
+      }
+    }
+  })
+
+  useSearch(
+    adminUserQuery,
+    {
+      search
+    },
+    search
+  )
+
+  const createRows = (rows: readonly UserType[]): UserRowType[] =>
+    rows.map((row) => {
+      return {
+        select: (
+          <Checkbox
+            value={selectedUsers.value.findIndex((invite) => invite.id === row.id) !== -1}
+            onChange={(value) => {
+              if (value) selectedUsers.merge([row])
+              else selectedUsers.set((prevInvites) => prevInvites.filter((invite) => invite.id !== row.id))
+            }}
+          />
+        ),
+        id: row.id,
+        name: row.name,
+        avatar: <AvatarImage src={row.avatar.thumbnailResource?.url || ''} name={row.name} />,
+        accountIdentifier: <AccountIdentifiers user={row} />,
+        lastLogin: row.lastLogin
+          ? new Date(row.lastLogin).toLocaleString('en-us', {
+              year: 'numeric',
+              month: 'short',
+              day: 'numeric',
+              hour: 'numeric',
+              minute: 'numeric'
+            })
+          : '-',
+        isGuest: row.isGuest.toString(),
+        action: (
+          <div className="flex items-center justify-start gap-3">
+            <button
```

Comment:
I think we should use tailwind `<Button />` control for all instances of `<button>` in this file.

If there are various button styles, then we should handle it through variants prop.